site-packages/integralstor_utils/disks.py

```diff
@@ -591,25 +575,43 @@ def get_disk_info_all():
         return return_dict, None
 
 
+def is_mounted(disk_name):
+    """Given a disk name like sda, returns true if its mounted"""
+    try:
+        cmd = "cat /proc/mounts | grep %s" % disk_name
+        output, err = command.get_command_output(
+            cmd, shell=True, check_rc=False)
+        if err:
+            raise Exception(err)
+        elif output == []:
+            is_mounted = False
+        else:
+            is_mounted = True
+    except Exception, e:
+        return None, "Unable to check for device %s: %s" % (disk_name, str(e))
+    else:
+        return is_mounted, None
+
+
 def get_file_system_by_name(disk_name):
     """get file system type by providing disk name or partition name"""
     try:
         cmd = "blkid %s -s TYPE -o value" % disk_name
-        ftype_li, err = command.get_command_output(cmd, shell=True)
+        fstype_list, err = command.get_command_output(cmd, shell=True)
         if err:
             raise Exception(err)
-        if len(ftype_li) == 1:
-            format_type = ftype_li[0]
+        if len(fstype_list) == 1:
+            fs_type = fstype_list[0]
         else:
             raise Exception(
                 "It looks like you have multiple partitions on %s \nenter the partition name (eg. /dev/sdx1)" % disk_name)
```

Comment:
Always use please.  - "It looks like you have multiple partitions on %s. Please enter the partition name (eg. /dev/sdx1)